tidy3d/components/simulation.py

```diff
@@ -5326,3 +5624,91 @@ def from_scene(cls, scene: Scene, **kwargs) -> Simulation:
         )
 
     _boundaries_for_zero_dims = validate_boundaries_for_zero_dims()
+
+    def _make_pec_frame(self, obj: Union[ModeSource, InternalAbsorber]) -> Structure:
+        """Make a pec frame around a mode source."""
```

Comment:
would be good to add in the docstring (also docstring of Simulation absorber fields, and absorber docstring) that both PEC frame and backing are added.